libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/ImageUtils.java

```diff
@@ -495,7 +495,7 @@ private static boolean resizeImageAndWriteToStream(Context context,
         }
 
         Bitmap.CompressFormat fmt;
-        if (fileExtension != null && fileExtension.equalsIgnoreCase("png")) {
+        if (fileExtension != null && fileExtension.contains("png")) {
```

Comment:
why this change? can a PNG file be any other thing than `png`? Or the other way around, couldn't this change bring false positives in?